### ISSUE TITLE
Not workin on Nginx?

### ISSUE THREAD

**Issue by tiotrom** [jodacame/youtube-music-engine#19]
Was wondering...

**Comment by FDiskas** [comment]
If you see `.htaccess` then it means - apache - so in your case you need to convert it to nginx rules.

**Comment by tio-trom** [comment]
I see. It seems quite a lot of work then. I am not sure I would be able to do it.

**Comment by FDiskas** [comment]
Actually this is so outdated. It will not work any way. Youtube api changed, no impossible to play without displaying a video. Also music search api also changed. Impossible to search. Also Facebook api changed. Impossible to login.
Good luck. If you manage to solve thous issues please make a pull request. ;)

**Comment by tio-trom** [comment]
My install still works ok for most part https://www.musikwave.com/ but is on Apache. Search works, playing songs works, and so forth.

**Comment by FDiskas** [comment]
last.fm is preventing images to be displayed https://www.musikwave.com/artist/Beck

**Comment by tio-trom** [comment]
Yes there are some errors and some api limitations. But the website is functional since 2014 or so. I use it, and a lot of other people. It is unfortunate that this script is not developed anymore...it is a very handy website to listen to music.
